### ISSUE TITLE
Auraed panic when `free` a cell that does not exist

### ISSUE THREAD

**Issue by krisnova** [aurae-runtime/aurae#145]
Steps to reproduce:

1. Start `auraed` 

2. Run the following typescript:

```typescript
await cells.free(<runtime.FreeCellRequest>{
    cellName: "non-existent-cell"
});
```

3. Observe `auraed` logs:

```
16:25:22 [INFO] Starting Aurae Daemon Runtime...
16:25:22 [INFO] Register Server SSL Identity
16:25:22 [INFO] Validating SSL Identity and Root Certificate Authority (CA)
16:25:22 [INFO] User Access Socket Created: /var/run/aurae/aurae.sock
16:25:31 [INFO] CellService: free() cell_name="sleeper-cell"
thread 'tokio-runtime-worker' panicked at 'find cell_name in cgroup_table', auraed/src/runtime/mod.rs:122:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by dmah42** [comment]
looks like an `except` snuck back in. got it.

**Comment by dmah42** [comment]
i can fix this in #144
